src/coordinate_transformations.jl

```diff
@@ -0,0 +1,91 @@
+mutable struct CRS2CRS <: CoordinateTransformations.Transformation
+    rep::Ptr{Cvoid}
+    direction::PJ_DIRECTION
+end
+
+function CRS2CRS(source::String, target::String, direction::PJ_DIRECTION = PJ_FWD; area = C_NULL, normalize = true)
+
+    if direction == PJ_IDENT || source == direction
+        return IdentityTransformation()
```

Comment:
Just a note that this makes `CRS2CRS` a little type unstable. I'm not sure whether that's worth it or not?

Reply:
Yeah, I wasn't sure about this myself.  We could definitely get rid of it.